cmd/ore/aws/upload.go

```diff
@@ -122,6 +123,26 @@ func defaultBucketURL(urlPrefix, imageName, board, file, region string) (*url.UR
 	return s3URL, nil
 }
 
+type qemuInfo struct {
+	VirtualSize uint64 `json:"virtual-size"`
+}
+
+// We used to hardcode 8GB for Container Linux, but let's support
+// differently-sized images.
+func getVMDKSizeGB(path string) (uint32, error) {
+	out, err := exec.Command("qemu-img", "info", "--output=json", path).Output()
```

Comment:
This is called unconditionally for both VMDK and raw images and will work correctly for both, but the function name implies otherwise.